src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+$day-size: 100px;
+$day-gap: 1px;
+$days-names: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+
 body {
   margin: 0;
 }
+
+.calendar {
+  max-width: 7 * $day-size + 10px;
```

Comment:
<img width="746" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/49764881/5edd4cd8-797e-467e-9791-0df0866e2503">

Also for variables